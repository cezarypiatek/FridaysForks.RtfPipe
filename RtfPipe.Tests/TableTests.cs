using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace RtfPipe.Tests;

[TestClass]
public class TableTests
{
  [TestMethod]
  public void marge_scenario_1() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_2() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_3() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_4() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_5() => this.VerifyCurrentScenario();
  
  [TestMethod]
  public void marge_scenario_6() => this.VerifyCurrentScenario();
  
  [TestMethod]
  public void marge_scenario_7() => this.VerifyCurrentScenario();
}
