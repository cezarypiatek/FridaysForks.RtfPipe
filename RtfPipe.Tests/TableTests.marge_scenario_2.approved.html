<div style="font-size:12pt;font-family:&quot;Times New Roman&quot;, serif;">
    <table style="line-height:1.3;font-family:Aptos, sans-serif;font-size:11pt;border-spacing:0;box-sizing:border-box;margin:0 0 0 0.3px;">
        <colgroup>
            <col style="width:154px;" />
            <col style="width:154px;" />
            <col style="width:154.1px;" />
            <col style="width:154.1px;" />
        </colgroup>
        <tr style="text-align:left;">
            <td rowspan="2" style="vertical-align:top;border:1px solid;padding:0 7.2px;">&nbsp;
            </td>
            <td style="vertical-align:top;border:1px solid;padding:0 7.2px;">&nbsp;
            </td>
            <td colspan="2" style="vertical-align:top;border:1px solid;padding:0 7.2px;">&nbsp;
            </td>
        </tr>
        <tr style="text-align:left;">
            <td style="vertical-align:top;border:1px solid;padding:0 7.2px;">&nbsp;
            </td>
            <td rowspan="2" style="vertical-align:top;border:1px solid;padding:0 7.2px;">&nbsp;
            </td>
            <td style="vertical-align:top;border:1px solid;padding:0 7.2px;">&nbsp;
            </td>
        </tr>
        <tr style="text-align:left;">
            <td style="vertical-align:top;border:1px solid;padding:0 7.2px;">&nbsp;
            </td>
            <td style="vertical-align:top;border:1px solid;padding:0 7.2px;">&nbsp;
            </td>
            <td style="vertical-align:top;border:1px solid;padding:0 7.2px;">&nbsp;
            </td>
        </tr>
    </table>
 </div>